packages/block-library/src/tag-cloud/editor.scss

```diff
@@ -7,7 +7,6 @@
 	span {
 		display: inline-block;
 		margin-left: 5px;
-		color: $gray-700;
 		text-decoration: none;
 	}
 }
```

Comment:
What about the rest of the styles in this file, are they needed?

Reply:
Feels some of them could be considered to be moved to `style.css` to have a slightly more opinionated block in terms of spacing.

Reply:
Yeah, I think the rest of the styles here might make sense to apply to the front-end actually. 

I definitely agree with removing the gray color, but I think the non-underlined number we have in the backend is a more polished visual treatment. I'd like to see that applied in both places.

Reply:
Good idea, PR updated...